### PR TITLE
add ball shape searchlight

### PR DESCRIPTION
**Pull request by lcnature**

### CODE REVIEW

**Comment by mjanderson09** [comment]
Looks good!

(Pdb) print(b.mask_[3,:])
[[False False False False False False False False False False False]
 [False False False  True  True  True  True  True False False False]
 [False False  True  True  True  True  True  True  True False False]
 [False  True  True  True  True  True  True  True  True  True False]
 [False  True  True  True  True  True  True  True  True  True False]
 [False  True  True  True  True  True  True  True  True  True False]
 [False  True  True  True  True  True  True  True  True  True False]
 [False  True  True  True  True  True  True  True  True  True False]
 [False False  True  True  True  True  True  True  True False False]
 [False False False  True  True  True  True  True False False False]
 [False False False False False False False False False False False]]

File: brainiak/searchlight/searchlight.py [hunk 1]
```diff
@@ -95,6 +95,32 @@ def __init__(self, rad):
                         self.mask_[r1, r2, r3] = True
 
 
+class Ball(Shape):
+    """Diamond
```
Comment (mihaic) [inline brainiak/searchlight/searchlight.py]: "Ball searchlight shape"?

**Comment by mihaic** [comment]
@lcnature, regardless of your off-by-one question (@mjanderson09?), please add unit tests for the new `Ball` class to pass the coverage check.

**Comment by lcnature** [comment]
@mihaic Done. Thanks!

**Comment by mihaic** [comment]
Thanks, @lcnature. I am sure your `Ball` shape will be very popular.

**Pull request merged**